scripts/helmInit.sh

```diff
@@ -9,6 +9,49 @@
 export PATH=$PATH:/usr/local/bin
 export HOME=/root
 export KUBECONFIG=/home/ec2-user/.kube/config
+
```

Comment:
I'm not suggesting we change this now but this doesn't feel like the right place for this, this script is supposed to be for initialising helm, maybe we need an additional script/command for cluster initialisation tasks?